### PR TITLE
Add Intellij files to .gitignore.

### PR DESCRIPTION
**Pull request by abayer**
cc @reviewbybees esp @kohsuke @oleg-nenashev @jglick

### CODE REVIEW

**Comment by andresrc** [comment]
:bee:

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 target
+**/*.iml
```
Comment (johnou) [inline .gitignore]: *.iml should suffice?

**Comment by jglick** [comment]
:bee:

File: .gitignore (hunk 1 above)
Comment (johnou) [inline .gitignore]: @jglick thanks for ignoring me completely.

**Pull request merged**